swig/python2/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+if(ENABLE_PYTHON)
+	include(${SWIG_USE_FILE})
+	set(PYTHON_COMPILED_FILES openscap.pyc openscap.pyo openscap_api.pyc openscap_api.pyo)
+
+	include_directories(${PYTHON_INCLUDE_PATH})
+
+	include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+	set(CMAKE_SWIG_FLAGS "-module" "openscap_py")
```

Comment:
you use CMAKE_SWIG_FLAGS here but in the perl one you used PROPERTIES SWIG_FLAGS. Is that intentional?

Reply:
@mpreisler ok, I'll try to make it more consistent.